torch/lib/c10d/ProcessGroupGloo.hpp

```diff
@@ -27,6 +27,10 @@
 
 namespace c10d {
 
+// Default timeout for operations against ProcessGroupGloo (30 mins)
+constexpr std::chrono::milliseconds kProcessGroupGlooDefaultTimeoutMillis =
```

Comment:
This creates inconsistency with nccl PG?

https://github.com/pytorch/pytorch/blob/ee1ddcef8d2994b59c9d7aeae8f0dd455ee6e469/torch/lib/c10d/ProcessGroupNCCL.cpp#L130-L135

Reply:
Good point, does it make sense for NCCL and Gloo to be the same as well? I think we should, the timeout is controlled and always provided as a default in Python, anyways.